### ISSUE TITLE
[QUESTION] Fastify already includes pino, and I want to configure it on Adapter level, and use this config for logger

### ISSUE THREAD

**Issue by julestruong** [iamolegga/nestjs-pino#1238]
I don't quite understand the explanation on why i should not use `useExisting` .
Can you rephrase it ? or explain it better to me ?

**Comment by iamolegga** [comment]
If you don't need startup logs and out of request context logs to use the same config that you are already provide in fastify then you are good to go. This notion is only about either duplicating config for those cases or have inconsistent config for those cases

**Comment by julestruong** [comment]
Ok let me explain my problem.

I use genReqId in both loggers.

I want the same reqId to be set. How do i do that ? 
All the application context logs are logged with nestjs-pino EXCEPT one : https://github.com/Eomm/mercurius-logging
I need this to be able to log the graphql query.

**Comment by iamolegga** [comment]
I'm not into fastify plugins, sorry, but I believe `useExisting` should work for you. There is [test](https://github.com/iamolegga/nestjs-pino/blob/master/__tests__/use-existing.spec.ts) that checks that passed to fastify params are used for logging from request context
